### PR TITLE
Changed lang translation file format/functions that read these files

### PR DESCRIPTION
**Pull request by chouske**
(Intern assignment) Seira requested that instead of having multiple language folders(en, jp) with json translation files to instead of one set of json translation files with all the language translations within them. I updated a function to read from this new file format and I did test using i18next.t in the console to make sure this new format does translate

### CODE REVIEW

File: src/core/translation/i18n.ts [hunk 1]
```diff
@@ -21,3 +24,5 @@ i18next
       escapeValue: false,
     },
   });
+//console.log(generateTranslationFile('en'));
```
Comment (satanmourner) [inline src/core/translation/i18n.ts:27]: remove comments please

File: src/core/translation/locales/translation.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import languages from './languages.json';
+import skills from './skills.json';
+import socialCauses from './socialCauses.json';
+
+export function generateTranslationFile(theLanguage) {
+  //return Object.assign({}, skills, socialCauses, languages);
```
Comment (satanmourner) [inline src/core/translation/locales/translation.ts:6]: remove comments please

**Review by satanmourner — CHANGES_REQUESTED** [review]

File: src/core/translation/locales/translation.ts (hunk 2 above)
Comment (chouske) [reply src/core/translation/locales/translation.ts]: > remove comments please

Ok I removed all four comments in my latest push.

**Comment by satanmourner** [comment]
@marjan-ara could you please check it too, thanks

**Comment by marjan-ara** [comment]
@satanmourner @chouske 
These changes are OK, but what's the purpose of this PR? I think just translating SKILLS and SOCIAL_CAUSES to another language won't help us that much. If we are going to translate all pages, we have to define variables page by page.

File: src/core/translation/locales/translation.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import socialCauses from './socialCauses.json';
+
+export function generateTranslationFile(theLanguage) {
+  const translationFile = {};
+  const combinedTranslations = Object.assign({}, skills, languages, socialCauses);
+  for (const key in combinedTranslations) {
+    translationFile[key] = combinedTranslations[key][theLanguage];
+  }
+  return translationFile;
+}
```
Comment (marjan-ara) [inline src/core/translation/locales/translation.ts:12]: These changes are OK and the PR can be merged, but what's the purpose of this PR? I think just translating SKILLS and SOCIAL_CAUSES to another language won't help us that much. If we are going to translate all pages, we have to define variables page by page.

Comment (seira-yun) [reply src/core/translation/locales/translation.ts:12]: @chouske 

Marjan is right. We need to define variables for each page. And all the variables should be in the i18n file with correspending translations.

**Pull request closed without merging**